### ISSUE TITLE
Doesn't work with Whatsapp Web audios

### ISSUE THREAD

**Issue by WilliamTael** [vzze/volume-control#2]

**Comment by vzze** [comment]
Could you give a more detailed explanation?

**Comment by WilliamTael** [comment]
The extension won't even show WhatsApp web on its menu. Doesn't change even
when I refresh the page.

Em ter., 28 de mar. de 2023 07:15, edi ***@***.***> escreveu:

**Comment by vzze** [comment]
The extension hides tabs that don't currently have any audio or video source, did the WhatsApp menu have any audio or video source? If no that is the reason for it not showing up, otherwise I'll look into it.

**Comment by WilliamTael** [comment]
It did. Audio voice notes. Still, it's not being detected.

Em qua., 29 de mar. de 2023 05:44, edi ***@***.***> escreveu:

**Comment by vzze** [comment]
After testing, I've concluded it is not possible to access the audio levels on Whatsapp for any kind of message sent be it video or audio voice messages. 

That is because of Whatsapp's policy: `Messages to *** are end-to-end encrypted. No one outside of this chat, not even WhatsApp, can read or listen to them. Click to learn more.`

The important part of the policy is `not even WhatsApp, can read or listen to them` which means not even the company itself can access the audio much less my own extension.

To further prove this point I tried brute force searching for the audio and it came up empty.
![image](https://user-images.githubusercontent.com/74437706/229306389-fd29f196-8017-4f6f-9b95-2e05b304fc8a.png)

I am sorry for not being able to provide a satisfying solution to your problem, but it is out of my hands.
